lib/bridge.js

```diff
@@ -94,32 +155,27 @@ Bridge.prototype.list = function list () {
  *
  * @memberOf Bridge#
  * @param {Number}    nukiId              the id of the nuki
- * @param {Boolean}   [doNotValidate]     if true, will not validate existance (by calling the list endpoint of the bridge)
  * @returns {Promise<Nuki>}
  */
-Bridge.prototype.get = function get (nukiId, doNotValidate) {
+Bridge.prototype.get = function get (nukiId) {
   var self = this;
 
-  if (doNotValidate) {
-    return Promise.resolve(new Nuki(self, nukiId, {delayBetweenRequests: self.delayBetweenRequests}));
-  }
-
   return this
     ._request('list')
     .then(function validateNuki (nukis) {
       if (!Array.isArray(nukis)) {
         throw new Error('did not receive a list of nukis');
       }
 
-      const found = nukis.some(function findNuki (nuki) {
-        return nuki.nukiId === nukiId;
+      const deviceType = nukis.some(function findNuki (nuki) {
```

Comment:
This is not correct.

Array.prototype.some returns true or false, not the first non falsy value like we would need here.

You could use .find and get the deviceType of the returned nuki, if one got found. It's available since node 4

Reply:
Yes, `find` is correct here. Thanks.

Reply:
The code is still wrong.  See this line: https://github.com/Mik13/nuki-bridge-api/pull/17/files#diff-3cde3a4ec5c47a534bed8986fd8a75cfR168

You return the deviceType, but the find does not return the type, but the object where a truthy value was returned.

Reply:
Thanks for that remark. I fixed it now.